frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -675,3 +675,142 @@ export const GRAPH_AXIS_SETTINGS = {
   // DEPRECATED" replaced with "label" series setting
   "graph.series_labels": {},
 };
+
+export const SANKEY_DATA_SETTINGS = {
```

Comment:
this file `graph.js` is more for settings that are used for many charts e.g. Line, Area, Bar, Scatter, Waterfall. But this settings is only used for a single chart. You could inline it inside the component file or create another settings file alongside the component file instead.

e.g. https://github.com/metabase/metabase/blob/a48893b44db7905f5e1165e84a0a0032992938f9/frontend/src/metabase/visualizations/visualizations/RowChart/utils/settings-definitions.js#L5

Reply:
I've moved it to the Sankey.jsx file.